android/src/main/kotlin/top/kikt/imagescanner/core/utils/DBUtils.kt

```diff
@@ -436,6 +436,7 @@ object DBUtils : IDBUtils {
         cursor.use {
             if (cursor.moveToNext()) {
                 val targetPath = cursor.getString(0)
+                targetPath.checkDirs()
```

Comment:
Can you apply the fix to other DBUtils too?

Reply:
sure, give me a minute

Reply:
I tested two Android 10 devices, Huawei mateX and Google Pixel3 , and it works ok with the `saveVideo` method, meanwhile the other DBUtils has the different implement, so I think it is no need to apply the fix to them

Reply:
I think it's necessary because most of feedbacks are using devices with Android 10 and 11, which also require a path check before the insertion.

Reply:
done.